tests/helpers.test.ts

```diff
@@ -170,26 +170,52 @@ describe('ModuleHelpers', () => {
       expect(mockNormalize).toBeCalledWith(settings.pathToJest)
     })
 
-    it('defaults to "node_modules/.bin/jest" when Jest is locally installed', () => {
-      const expected = 'node_modules/.bin/jest'
+    describe('os platform: linux', () => {
```

Comment:
Even though it is the easiest solution, I dislike to duplicate the whole test logic for different operating systems. In my opinion it would be better to run those tests against a generic extension.

Reply:
I understand and I guess that's another change that will come with your PR :)